src/utils/metrics.ts

```diff
@@ -0,0 +1,94 @@
+import client from "prom-client";
+
+export class MetricsService {
```

Comment:
is there any added benefit from wrapping the metrics in a Class? I would just export them directly

Reply:
Nothing explicit unless we want to at some stage allow for metrics to be specified on a different port to the API, and therefore would have an object here that instantiates a `HttpServer`.